system_modules/napaudio/src/audio/resource/audiofileresource.cpp

```diff
@@ -32,16 +46,18 @@ namespace nap
 			float sampleRate;
 			if (readAudioFile(mAudioFilePath, *getBuffer(), sampleRate, errorState))
 			{
-				auto serviceSampleRate = mAudioService->getNodeManager().getSampleRate();
-				if( serviceSampleRate != sampleRate){
-
-					nap::Logger::info("Resampling audio file: %s", mAudioFilePath.c_str());
-					if(!resampleSampleBuffer(*getBuffer(), sampleRate, serviceSampleRate, 2, errorState))
-						return false;
-					
-					sampleRate = serviceSampleRate;
-
-
+				if(mResample)
+				{
+					auto serviceSampleRate = mAudioService->getNodeManager().getSampleRate();
+					if( serviceSampleRate != sampleRate)
+					{
+	
+						nap::Logger::info("Resampling audio file: %s", mAudioFilePath.c_str());
```

Comment:
Better to use `nap::Logger::debug()` instead of info. Ensures the info isn't shown unless debug mode is enabled.